pypesto/visualize/sampling.py

```diff
@@ -1308,9 +1308,6 @@ def sampling_1d_marginals(
     # fig, ax = plt.subplots(nr_params, figsize=size)[1]
     for idx, par_id in enumerate(param_names):
         if plot_type == 'kde':
-            # bw_method argument is deprecated... FutureWarning:
```

Comment:
what happened here?

Reply:
this is some old code, so I removed it, it is hinting towards the `bw` argument, which was used previously, and now is replaced by bw_method`